src/main/java/TaskList.java

```diff
@@ -5,7 +5,11 @@ public class TaskList {
     public void listTasks() {
         int i = 1;
         for (Task task:tasklist) {
-            System.out.println(" " + i + ". " + task.taskName);
+            if (task.completed) {
+                System.out.println(" " + i + ".[X] " + task.taskName);
```

Comment:
Maybe we can consider adding a ```public String getStatus()``` method to the ``` Task class``` which will help output "X", which may be better than hard-coding :)